### DISCUSSION TITLE
IRRemote: IRSend with known Protocoll and 120 bit data will not work with 9.3.1

### DISCUSSION THREAD

**Discussion by ZathrasIV** [arendst/Tasmota#13578] [Q&A] [answered]
Hi, I'm knew here and have a problem with tasmota-ir 9.3.1

In the console I can receive valid packages of my Air Conditioner RC:
_{"IrReceived":{"Protocol":"MIRAGE","Bits":120,"Data":"0x5674000013C040000000000000002A","Repeat":0}}_

So the protocoll seems to be known, and all bits are there.
Trying to send out the same command will not work:
_IRSend {"Protocol":"MIRAGE","bits":120,"Data":0x5674000013C040000000000000002A}_

tasmota-ir does work so with my other RCs

I know, 120Bit data is more, than in the documentation states but from the forum here (post#12781) I read, that 114 Bits 
are fine. 
And at least I can receive them correctly!

A workaround with raw data is working, but not suitable for me, as each IR data package needs to be calculated
by SW in advance- which will be painfull in raw.

Any hint welcome, thanx

**Comment by s-hadinger** [comment]
Sending Hex numbers is limited to 64 bits unfortunately. You can use raw or hope à higher level interface is added in IRREMOTEESP8266

**Comment by Jason2866** [comment, marked as answer]
You can open a feature request here https://github.com/crankyoldgit/IRremoteESP8266

**Comment by ZathrasIV** [comment]
Thank you for your fast reply. 
Digging through the feature requests, I found, that my A/C driver is nearly finished (#1573)
So I will wait with my implementation ;-)

Great community here!!!
